spec/features/admin_creates_quiz_spec.rb

```diff
@@ -8,10 +8,31 @@
     click_on "New Quiz"
   end
 
-  scenario "successful quiz creation" do
+  scenario "successful quiz creation", js: true do
+    skip
     fill_in "Title",       with: "Week 1 - Ruby basics"
     fill_in "Description", with: "Quiz focused on Ruby methods and iterations"
     uncheck "Published"
+
+    click_on "Add Question"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.